tests/inte_test/test_dssm.py

```diff
@@ -26,11 +30,30 @@ def inte_test_dssm():
     # do pre-processing.
     dssm_preprocessor = preprocessor.DSSMPreprocessor()
     processed_train = dssm_preprocessor.fit_transform(train, stage='train')
+    # the dimension of dssm model is the length of tri-letters.
+    dim_triletter = processed_train.context['dim_triletter']
     # generator.
     generator = generators.PointGenerator(processed_train)
-    # TODO GENERATOR
     X, y = generator[0]
-    print(X)
+    # Create a dssm model
+    dssm_model = models.DSSMModel()
+    dssm_model.params['input_shapes'] = [(dim_triletter, ), (dim_triletter, )]
+    dssm_model.guess_and_fill_missing_params()
+    dssm_model.build()
+    dssm_model.compile()
+    dssm_model.fit([X['text_left'], X['text_right']], y)
+    dssm_model.save('.tmpdir')
+
+    # testing
+    processed_test = dssm_preprocessor.fit_transform(test, stage='test')
```

Comment:
Here,  the testing phase can be implemented in an separate test function. Thus, we can also test `load dssm_preprocessors`.